lightkurve/targetpixelfile.py

```diff
@@ -197,12 +198,10 @@ def __repr__(self):
         return('KeplerTargetPixelFile Object (ID: {})'.format(self.keplerid))
 
 
-    def find_stars(self, catalog=None, radius=0.5):
+    def query_catalog(self, catalog=None, radius=0.5, seperation_factor=12):
```

Comment:
I'm not sure we want this separation factor, as it's redundant with radius. Why would I set radius and separation factor?

(It also needs a better name, perhaps tolerance.)